src/style.css

```diff
@@ -1 +1,49 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-direction: row;
+  margin: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(../src/images/star-active.svg);
+}
```

Comment:
The selector `.stars--0 .stars__star:nth-child(-n + 0)` is incorrect because it selects no elements. The `nth-child(-n + 0)` will not match any elements since it effectively means 'select up to the 0th child'. Consider revising this selector to achieve the desired effect, possibly by removing it or adjusting the logic.